package.json

```diff
@@ -29,7 +29,7 @@
     "prismarine-item": "^1.0.1",
     "prismarine-recipe": "^1.0.1",
     "prismarine-windows": "^1.1.0",
-    "protodef": "^1.6.7",
```

Comment:
Why ?

1. Unrelated with this review
2. Not needed because of the ^

Reply:
Just merged two of the update branches. I think that dependencies should be updated. Especially minecraft-data ;)